RockPaperScissors.c

```diff
@@ -5,68 +5,78 @@
 
 int main(void)
 {
-    
     // Initialize random number generator
     srand(time(NULL));
-    int n;
-    int x = 0;
-    int Rock = 1; 
-    int Paper = 2; 
-    int Scissors = 3; 
+
+    // Allocate space for variables
+    int userChoice;
+    int aiChoice;
+    int round = 0;
+
+    // Initialize constants
+    const int Rock = 1; 
+    const int Paper = 2; 
+    const int Scissors = 3; 
```

Comment:
Constants should be marked `const`. This way you won't reassign these variables accidentally. 

You're also promising the C compiler that you'll never reassign these variables -- so the C compiler can optimize around that assumption and make your code faster. Free performance boost that can add up in larger pieces of code.